### PR TITLE
Update env set up with devenv

### PR DESCRIPTION
**Pull request by kddubey**
Current setup didn't work for me. Pretty sure this is the intended setup, as I think I can run sentry and seer locally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,12 +21,29 @@ pyenv local 3.11
...
-3. Create a `.env` file based on `.env.example` and set the required values
-4. (Optional) Add `SENTRY_AUTH_TOKEN=<your token>` to your `.env` file
+1. Install [devenv](https://github.com/getsentry/devenv?tab=readme-ov-file#devenv)
+2. Run
+    ```
+    devenv bootstrap
+    ```
+3. Run
+    ```
+    devenv fetch sentry
```
Comment (kddubey) [inline README.md:34]: Need to do this first to install certain dependencies

File: README.md [hunk 2]
```diff
@@ -21,12 +21,29 @@ pyenv local 3.11
...
+    ```
+4. After restarting your shell, run
+    ```
+    devenv fetch getsentry/seer
+    ```
+5. cd to the repo root
+6. Run
+    ```
+    python -m pip install -r requirements-precommit.txt
```
Comment (kddubey) [inline README.md:43]: I'm not sure how to add this to the [devenv config](https://github.com/getsentry/seer/blob/1338b499724dcf500023f44778c1345b26924761/devenv/config.ini#L4)

**Comment by aayush-se** [comment]
Thanks for adding in these updates, though had some questions here. These changes seem to be for Sentry related environment installation not Seer related. 

@Mitan did you have to do any of these steps when setting up sentry on your end since you set up seer + sentry most recently?

**Comment by kddubey** [comment]
This was probably something wrong I did during my installation. Current setup is fine and independent of sentry setup

**Pull request closed without merging**